index.html

```diff
@@ -4580,6 +4580,319 @@ <h1>Current Issues</h1>
+    representation of the <a>DID subject</a>. For example, a data schema that
+    needs a persistent, cryptographically verifiable identifier could be
+    assigned a <a>DID</a>, and its <a>DID document</a> could be used as a
+    standard way to retrieve a representation of that schema.
+    </p>
+
+    <h2>Can existing web resources also be assigned DIDs?</h2>
+    <p>
+    Yes, if the controller of a web page or any other web resource wants to
+    assign it a persistent, cryptographically verifiable identifier, the
+    controller can give it a <a>DID</a>. For example, the author of a blog
+    hosted by a blog hosting company (under that hosting company’s own URL)
+    could create a <a>DID</a> for the blog. In the <a>DID document</a>, the
+    author can include an alsoKnownAs property pointing to the current URL of
+    the blog:
+    </p>
+    <code>
+        “alsoKnownAs”: [“https://myblog.blogging-host.example.com/home”]
+    </code>
+    <p>
+    If the author subsequently moves the blog to a different hosting company
+    (or to the author’s own domain), the author can update the <a>DID document</a>
+    to point to the new URL for the blog:
+    </p>
+    <code>
+        “alsoKnownAs”: [“https://myblog.example.com/”]
+    </code>
+    <p>
+    The <a>DID</a> effectively adds a layer of indirection for the blog URL.
+    This layer of indirection is under the control of the author instead of
+    under external administrative authority such as a blog hosting company. This
+    is how <a>DIDs</a> can effectively function as
+    <a href="https://tools.ietf.org/html/rfc8141">URNs (Uniform Resource
+    Names)</a>—persistent identifiers for information resources whose network
+    location may change over time.
+    </p>
+  </section>
+
+  <section class="appendix">
+    <h1>
+    DID Controllers and DID Subjects
+    </h1>
+    <p>
+    To avoid confusion about the relationship between <a>DID controllers</a> and
+    <a>DID subject</a>s, the DID Working Group has found it helpful to classify
+    <a>DID subject</a>s into two disjoint sets based on their relationship to
+    the <a>DID controller</a>.
+    </p>
+    <h2>Set 1: The DID subject <em>is</em> the DID controller</h2>
+    <p>
+    The first case, shown in figure 4, is the common scenario where the
+    <a>DID subject</a> is also the <a>DID controller</a>. This is the case when
+    an individual or organization creates a <a>DID</a> to self-identify.
+    </p>
+        <figure id="controller-subject-equivalence">
+          <img style="margin: auto; display: block; width: 75%;"
+            src="diagrams/figure-b.1-controller-and-subject-equivalence.png" alt="
+            Diagram showing the same graph model as figure 2 except with an
+            equivalence arc from the DID subject to the DID controller.
+          " >
+          <figcaption>
+              The <a>DID subject</a> is the same entity as the <a>DID controller</a>
```

Comment:
This statement still feels a bit off to me. Curious if you're open to saying:

"The DID Subject is the same entity as the DID Controller with the assistance of additional software"

This nuance has become more apparent to me as we've looked at SSI from a legal perspective and so my thinking is that adding this will help to align legal minds and technical minds a bit more in their mental models.

Reply:
I am not sure I fully grasp the distinction. If the DID Controller, while identifying the same entity as the DID subject, has a different DID as a value, then I can see what you mean. But I would expect that if I create the DID document for my own DID (ie, identifying me) then the value of `subject` and `controller` will be the same DID, in which case I am not sure how I would understand your statement...

Reply:
It feels odd to me to include mention of the "additional software" for the DID Controller.
If I rake the leaves in my yard, I don't say "I raked my leaves with the assistance of a rake," I say, "I raked my leaves."
I seems like pointing out that a DID Controller uses a tool to exert control of a DID is unnecessary. Of course there is a tool. I don't rake my leaves by picking them up one at a time with my hands, nor do I create a digital signature by counting on my fingers.
Just feels like a distinction without a difference.

Reply:
The difference comes when considering that the software is often not under the control of the Subject and is therefore capable of behaving in a superfluous behavior. While from a technical perspective it may appear logical to conjoin the two and say they're the same entity legal precedents have shown that there is a distinction between the actions taken using an identifier and the connection to the person. Therefore, what I'm arguing here is that we should not logically assume that because the software is behaving in a way that the user consented to that it will always be interpreted in that way.

As an example of the legal precedent set today in the US, in case [Malibu Media LLC v. John Doe](https://www.scribd.com/document/214110295/213923681-Gov-uscourts-flsd-434635-10-0?ad_group=111346X1569475X3149821452016d325c03d3ffaefdf395&campaign=SkimbitLtd&keyword=660149026&medium=affiliate&source=hp_affiliate) they state:

> however, Plaintiff has not shown how this geolocation software can establish the identity of the Defendant. There is nothing that links the IP address location to the identity of the person actually downloading and viewing Plaintiff’s videos, and establishing whether that person lives in this district. For example, when arguing that this IP address is not a coffee shop or open Wi-Fi network, Plaintiff points to the timing of the alleged infringement and the fact that the internet service provider typically provides internet to residences.

This distinction basically sets the precedent that while the DID Subject is a logical part of the the DID Controller (via clicking buttons to take an action), there is not grounds to link an identifier to a person because the identifier could logically be more than a single person especially in the case that the software is acting autonomously on behalf of the user.

Therefore, setting the expectation that the DID Controller CAN be the DID Subject, but will never solely be the DID Subject is an important distinction when trying to frame the mental model in a continuous way that meets both the expectations of the implementers and the people building laws and regulations to match together through this spec.

Reply:
@brentzundel @iherman @talltree does my argument above sway you guys at all on this topic?

Reply:
I won't fight it going in if others want it, but it feels unnecessarily pedantic to me, especially in a non-normative part of the spec.

Reply:
@kdenhartog Sorry to take so long to reply to this thread but I have been waiting for to gather all the suggestions on the Appendix before processing them all (which I'm doing now).

So, first off, yes, I understand the distinction you are making between the DID controller as an entity and the software used by the DID controller to effect control of the DID / DID document. However I don't think it makes it clearer to expand the definition of **DID controller** to include the software (I actually think that make the situation muddier).

However it **is** a good idea is to clearly state that the DID controller exerts control via software, and from an architectural perspective (and even a legal perspective), that software needs to be taken into account.

I will reflect that notion in my update to the Appendix text that I'm preparing right now. However I think your point about the separation between the DID controller as an entity (human, org, thing) and the software is important enough that you should do a PR to either add a separate subsection of Security Considerations to explain it — or see if it can be added to an existing subsection.